web/src/features/charts/bar-breakdown/BarElectricityBreakdownChart.tsx

```diff
@@ -33,7 +33,7 @@ interface BarElectricityBreakdownChartProps {
   ) => void;
   onProductionRowMouseOut: () => void;
   onExchangeRowMouseOver: (
-    mode: ElectricityModeType,
+    mode: ZoneKey,
     data: ZoneDetail,
     event: React.MouseEvent<SVGPathElement, MouseEvent>
   ) => void;
```

Comment:
This entire interface seem to be the exact same as the ` BarBreakdownEmissionsChartProps` interface in `BarBreakdownEmissionsChart.tsx` so we can probably deduplicate them.

Reply:
They are very close, but with slight differences:
<img width="491" alt="Screenshot 2023-02-24 at 20 21 13" src="https://user-images.githubusercontent.com/3296643/221272128-2ba82d50-f254-4848-a6c8-73e92da40e23.png">

We could extract the common part and extend from that, but I don't think it's worth abstracting it for just these two components :)